src/js/step.js

```diff
@@ -190,7 +190,7 @@ export class Step extends Evented {
   _createTooltipContent() {
     const content = document.createElement('div');
     const classes = this.options.classes || '';
-    const element = createFromHTML(`<div class='${classes}' data-id='${this.id}' id="step-${this.options.id}-${uniqueId()}"}>`);
+    const element = createFromHTML(`<div class='${classes}' data-shepherd-step-id='${this.id}' id="step-${this.options.id}-${uniqueId()}"}>`);
```

Comment:
I'm good w/ removing the `ID` attr, but since with multiple tours we can have shared ids, can we do something to make sure this value is unique?

Reply:
Then again, do we need uniqueness here? 

We were discussing having a `data-shepherd-active-tour` attribute on the body. If we implement that, someone could always use it as the context for querying unique steps -- even with shared ids.

Reply:
@chuckcarpenter @rwwagner90 To summarize, I'm thinking we merge this PR as-is, then open up another issue for adding `data-shepherd-active-tour` attribute and removing the extra `id` attribute on steps.

Reply:
@chuckcarpenter Where are you seeing that? I'm thinking we still need to implement it. Right now, it looks like we're just indicating the current step:

<img width="1440" alt="screen shot 2018-10-22 at 4 01 26 pm" src="https://user-images.githubusercontent.com/5483853/47315939-e2a53280-d613-11e8-99c3-395bcc21632b.png">

Reply:
you're right. It's in the BM frontend right now. 

```
// TODO: add this in Shepherd.js
document.body.setAttribute('data-shepherd-active-tour', tourName);
```

Reply:
that said, should the tour id be part of the ID for the step?

Reply:
@chuckcarpenter [I don't think it would be necessary](https://github.com/shipshapecode/shepherd/pull/282#discussion_r226879920).